pages/cart.css

```diff
@@ -0,0 +1,28 @@
+.cart-summary {
+  height: min-content;
+}
+@media only screen and (max-width: 768px) {
+  .cart-summary {
+    width: 80%;
+  }
+}
+@media only screen and (max-width: 1124px) {
+  .cart-card {
+    width: 100%;
+  }
+}
```

Comment:
Preferable to stick to most commonly used screen sizes for media queries

Reply:
Yup